### ISSUE TITLE
Fuel pump relay, while engine running, is being pulsed on/off

### ISSUE THREAD

**Issue by NMSTEC** [rusefi/rusefi#4244]

**Comment by rusefillc** [comment]
i can reproduce this issue on hellen miata 96 board

**Comment by rusefillc** [comment]
@mck1117 do you have any board where you could reproduce or shall I ship you e67?

**Comment by mck1117** [comment]
@rusefillc Does it repro running e67 firmware, or running miata firmware on some other fuel pump pin?

**Comment by rusefillc** [comment]
e67 FW on e67 HW
miata FW on miata HW

**Comment by rusefillc** [comment]
yep, this has no connection with E67. I wonder if @andreika-git can reproduce on hellen81 on H7

@BeerMoneyMotorsports should be careful about P18/P22 aux pin for now
